packages/playwright/src/index.ts

```diff
@@ -81,13 +83,11 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
       options.headless = headless;
     if (channel !== undefined)
       options.channel = channel;
-    options.tracesDir = tracing().tracesDir();
```

Comment:
I see that `_browserOptions` is also used for connect, and we are now missing `tracesDir` there. Perhaps it was not the best idea to plumb it separately, sorry.

Reply:
I've tangled `tracesDir` back into `_defaultLaunchOptions` :D Hoping that works.